interface/billing/sl_eob_search.php

```diff
@@ -1071,7 +1147,19 @@ function npopup(pid) {
 
 if (!$INTEGRATED_AR) SLClose();
 ?>
-
+<tr bgcolor='<?php echo $bgcolor ?>'>
+<td>Totals</td>
```

Comment:
Brady,
I have been going over and over this request. The translation would have to take place inside the JavaScript at line 502. Note that variable is placed by the JavaScript and not the PHP.  So is there an xlt for JavaScript?